pkg/httpmetrics/metrics.go

```diff
@@ -149,5 +149,5 @@ func (r *requestDurations) buildTimer(metricsFactory metrics.Factory, key record
 			"method": key.method,
 		},
 	})
-	return
+	return out
```

Comment:
We can remove the name "out" from the signature

Reply:
The defer function on top requires the named output as its assigning a value to it